ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -446,6 +446,8 @@ Add\ recipient = Add recipient
 phone\ number\ not\ unique = This number is already registered for contact {{name}}
 version.update.title = Update available
 version.update.description = An updated version of the website is available. Reload to get the latest version.
+database_closed.title = An Error Has Occurred
+database_closed.description = An unexpected error has occurred and the application must be reloaded.
```

Comment:
What do you think of these strings?

Reply:
I might change the title to "Unexpected error", but it's fine. Definitely change it from title case to sentence case to match the rest of the content.

Reply:
Updated title